renderdoc/driver/d3d11/d3d11_device2_wrap.cpp

```diff
@@ -74,7 +74,8 @@ void WrappedID3D11Device::GetResourceTiling(
   if(m_pDevice2 == NULL)
     return;
 
-  m_pDevice2->GetResourceTiling(pTiledResource, pNumTilesForEntireResource, pPackedMipDesc,
```

Comment:
There is a helper function `UnwrapResource` to help i.e.

`m_pDevice2->GetResourceTiling(UnwrapResource(pTiledResource), ...`

Reply:
It's Ok, you can add test first and I will change to use helper funciton